logos-codegen/src/mir.rs

```diff
@@ -10,12 +10,28 @@ use crate::error::{Error, Result};
 
 lazy_static! {
     /// DOT regex that matches utf8 only.
-    static ref DOT_UTF8: Hir = Hir::dot(false);
+    static ref DOT_UTF8: Hir = Hir::dot(Dot::AnyChar);
 
     /// DOT regex that matches any byte.
-    static ref DOT_BYTES: Hir = Hir::dot(true);
+    static ref DOT_BYTES: Hir = Hir::dot(Dot::AnyByte);
 }
 
+macro_rules! literal {
+    ($byte:expr) => {
+        hir::Literal(Box::new([$byte]))
+    };
+    (@char $c:expr) => {
+        hir::Literal(
+            $c.encode_utf8(&mut [0; 4])
+                .as_bytes()
+                .to_vec()
+                .into_boxed_slice(),
+        )
+    };
+}
+
+pub(crate) use literal;
```

Comment:
Is there a reason for the macro to be defined here, exported but only used in `ignore_flags.rs`?

Reply:
I though it was used in both modules, but it maybe is not the case anymore